src/Plugin/WebformElement/CivicrmOptions.php

```diff
@@ -152,13 +152,17 @@ public function prepare(array &$element, WebformSubmissionInterface $webform_sub
     $as_list = !empty($element['#extra']['aslist']);
     $is_multiple = !empty($element['#extra']['multiple']);
     $use_live_options = !empty($element['#civicrm_live_options']);
+    $data = [];
+    if ($webform_submission) {
+      $data = $webform_submission->getWebform()->getHandler('webform_civicrm')->getConfiguration()['settings']['data'] ?? [];
+    }
 
```

Comment:
Why if `$webform_submission` here?

Reply:
From param definition, it seems it can also arrive as NULL, hence avoiding any notice or fatal error in that case.